modules/sandboxed-containers-deleting-kataconfig-crd-cli.adoc

```diff
@@ -0,0 +1,41 @@
+//Module included in the following assemblies:
+//
+// *uninstalling-sandboxed-containers.adoc
+
+:_content-type: PROCEDURE
+[id="sandboxed-containers-deleting-kataconfig-CRD-cli_{context}"]
+= Deleting the `KataConfig` custom resource definition using the CLI
+
+The `KataConfig` custom resource definition lets you define the `KataConfig` CR. Delete the `KataConfig` custom resource definition from your cluster.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
```

Comment:
Not sure if you use variables from the common-attributes file for your docs but I think there is one for OpenShift, if you do.

Reply:
I'm not sure there's one for just "OpenShift", `{product-title}` evaluates to the full "OpenShift Container Platform", though maybe there should be one for just "OpenShift", or "OpenShift CLI"!

Reply:
I copied this sentence over and it appears in multiple places, so am leaving for now. However, I have sent a question to the primary writer of the project about creating an attribute for this.